public/app/ui/components/search/queryBuilder/QueryBuilderComponent.less

```diff
@@ -0,0 +1,74 @@
+.query-builder {
+    .group-conditions {
+        margin: @padding-small-vertical 0 @padding-small-vertical 30px;
+    }
+
+    .rules-list {
+        padding: 0 0 0 30px;
+        position: relative;
+
+        > * {
+            position: relative;
+
+            &::before,
+            &::after {
+                position: absolute;
+                left: -30px;
+                width: 30px;
+                height: ~"calc(50% + 6px)";
```

Comment:
This syntax means to keep the string intact when it goes into CSS.  If it was just written as `height: calc(50% + 6px);` then LESS would do the calculation at build time, which isn't what we want.  By doing it like this, the resulting CSS is the call to `calc`, not the static result of that call.